### ISSUE TITLE
Access key invalid error.

### ISSUE THREAD

**Issue by Amirali-Amirifar** [miladrahimi/outline-bridge-server#5]
Problem:
Upstream server is up, client still can connect but but for unknown reasons !!! bandwidth is super low and almost unusable (probably less than 5kbps).
The bridge server setup has been done with no errors. but when you change your access key IP to bridge server IP the `It seems like your access key is not valid. If this happens again, please contact your server admin.` Error shows up.
Bridge: ubuntu 20.4
Upstream Centos 9

**Comment by miladrahimi** [comment]
Please share (you can email me: info@miladrahimi.com) a real key and it's modifed version after changing hostname.

**Comment by Amirali-Amirifar** [comment]
I have emailed you the requested information.

Important note: I found out that whenever I use Irancell as ISP, the client app shows the mentioned error but using ADSL Internet the connection is almost flawless. Could they somehow deform the access key on the fly? 
I have used this repository before, and my last server is up and running with no errors on all ISPs! It's so confusing! Now when I'm trying to replicate this, this problem has come up.
